lib/drupal.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(grunt) {
   };
 
   module.majorVersion = function() {
-    var version = grunt.config('drupal.drupal-version') || '',
+    var version = process.env.GDT_DRUPAL_CORE || grunt.config('drupal.drupal-version') || '',
```

Comment:
Drush is already used to determine this. drupal.drupal-version is set by dumping `drush status --format=json` into grunt.config('drupal') earlier in this file.